test/integration/encode_decode_cycle_test.go

```diff
@@ -26,7 +26,10 @@ import (
 // encode-decode-encode loop which will detect lossy behavior in both directions.
 func TestEncodeDecodeEncodeCycleComparison(t *testing.T) {
 	// use second image for relationships
-	images := []string{"image-pkg-coverage", "image-owning-package"}
+	images := []string{
+		//"image-pkg-coverage",
+		"image-owning-package",
+	}
```

Comment:
Was this an intentional change? I don't think we should remove the `image-pkg-coverage` test

Reply:
^ nooo good catch - I'll bring that back it was just to isolate the image for testing

Reply:
I'll update the tests with the new config option too for integration